prowler/lib/check/models.py

```diff
@@ -146,6 +146,21 @@ def __init__(self, metadata):
         self.location = ""
 
 
+@dataclass
+class Check_Report_Kubernetes(Check_Report):
```

Comment:
```suggestion
class CheckReportKubernetes(Check_Report):
```

This is to start changing the class names to comply with Python naming standards.

Reply:
We will have to change all the naming in the other providers then...

Reply:
TODO added